topi/python/topi/cuda/vision.py

```diff
@@ -0,0 +1,99 @@
+# pylint: disable=invalid-name, unused-variable
+"""Schedule for vision operators"""
+from __future__ import absolute_import as _abs
+import tvm
+import topi
+from .. import tag
+from .. import generic
+
+@generic.schedule_reorg.register(["cuda", "gpu"])
+def schedule_reorg(outs):
```

Comment:
if there is no special schedule for reorg and shortcut, let us remove these schedule functions and directly add note in the compute to ask user to use schedule_injective